.github/workflows/github-action.yml

```diff
@@ -52,4 +52,6 @@ jobs:
       run: mix format --check-formatted
     - name: Run tests
       run: mix test
+    - name: run docker
+      run: docker-compose build
```

Comment:
not docker-compose. we need to build image. the command is `docker build ...`

Reply:
@Menubatenny Let's also split this into another job, not run after mix test

The docker build should only run if `mix test` job passes. See if you can configure github action in that way